### ISSUE TITLE
Upgrade notes

### ISSUE THREAD

**Issue by dolzenko** [sporkd/compass-html5-boilerplate#9]
I have just upgraded 0.3.0 => 0.3.2 and suddenly all the links turned virgin blue.
Obviously the cause is https://github.com/sporkd/compass-html5-boilerplate/commit/411db5 which moved link styling from `stylesheets/html5-boilerplate/_reset.scss` to `templates/project/partials/_page.scss`.

I guess currently the best way to watch out for such issues is `git log templates` whenever upgrading?

**Comment by sporkd** [comment]
Yes.  Sorry about this.  I moved a few lines out of the Compass library and into the user editable partials because Html5 Boilerplate now flags them as stuff that should be overridden.

I'll try and deprecate stuff like this in the future so as not to break things.  But for now you could simply add the following lines to your page partial.

https://github.com/sporkd/compass-html5-boilerplate/blob/v0.3.2/templates/project/partials/_page.scss#L8-22
